lib/omnibus/packager.rb

```diff
@@ -68,7 +68,15 @@ def for_current_system
         family = "solaris"
       end
       if klass = PLATFORM_PACKAGER_MAP[family]
-        klass.is_a?(Array) ? klass : [ klass ]
+        package_types = klass.is_a?(Array) ? klass : [ klass ]
+
+        if package_types.include?(APPX) &&
+          !Chef::Sugar::Constraints::Version.new(version).satisfies?('>= 6.2')
```

Comment:
yeah I had wondered the same but got `undefined local variable or method `platform_version' for Omnibus::Packager:Module` and also saw the solaris checks were using the longer form. I'll look to see why `platform_version` does not take. Maybe something to do with how the [packager module is pulled in.